src/main/java/org/folio/marc4ld/service/ld2marc/mapper/impl/agent/PersonFamilyMapper.java

```diff
@@ -0,0 +1,82 @@
+package org.folio.marc4ld.service.ld2marc.mapper.impl.agent;
+
+import static org.folio.ld.dictionary.PredicateDictionary.CREATOR;
+import static org.folio.ld.dictionary.PropertyDictionary.AFFILIATION;
+import static org.folio.ld.dictionary.PropertyDictionary.ATTRIBUTION;
+import static org.folio.ld.dictionary.PropertyDictionary.CONTROL_FIELD;
+import static org.folio.ld.dictionary.PropertyDictionary.DATE;
+import static org.folio.ld.dictionary.PropertyDictionary.EQUIVALENT;
+import static org.folio.ld.dictionary.PropertyDictionary.FIELD_LINK;
+import static org.folio.ld.dictionary.PropertyDictionary.LINKAGE;
+import static org.folio.ld.dictionary.PropertyDictionary.NAME;
+import static org.folio.ld.dictionary.PropertyDictionary.NAME_ALTERNATIVE;
+import static org.folio.ld.dictionary.PropertyDictionary.NUMERATION;
+import static org.folio.ld.dictionary.PropertyDictionary.TITLES;
+import static org.folio.ld.dictionary.ResourceTypeDictionary.CONCEPT;
+import static org.folio.ld.dictionary.ResourceTypeDictionary.FAMILY;
+import static org.folio.ld.dictionary.ResourceTypeDictionary.PERSON;
+import static org.folio.marc4ld.util.Constants.SPACE;
+import static org.folio.marc4ld.util.Constants.TAG_100;
+import static org.folio.marc4ld.util.Constants.TAG_700;
+import static org.folio.marc4ld.util.Constants.THREE;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.util.Map;
+import java.util.Set;
+import org.folio.ld.dictionary.ResourceTypeDictionary;
+import org.folio.ld.dictionary.model.Resource;
+import org.folio.ld.dictionary.model.ResourceEdge;
+import org.folio.marc4ld.service.dictionary.DictionaryProcessor;
+import org.marc4j.marc.MarcFactory;
+import org.springframework.stereotype.Component;
+
+@Component
+public class PersonFamilyMapper extends AgentMapper {
+
+  private static final Set<ResourceTypeDictionary> SUPPORTED_TYPES = Set.of(CONCEPT, PERSON, FAMILY);
```

Comment:
CONCEPT will not be part of the supported types. Only PERSON and FAMILY should be in this list.

Reply:
removed